processor/src/trace/virtual_bus/sum_check/prover/mod.rs

```diff
@@ -221,7 +222,7 @@ where
         // generate the last random challenge
         let round_challenge = coin.draw().map_err(|_| Error::FailedToGenerateChallenge)?;
         // fold each multi-linear using the last random challenge
-        mls.iter_mut().for_each(|ml| ml.bind(round_challenge));
+        mls.iter_mut().for_each(|ml| ml.bind_left(round_challenge));
```

Comment:
should either be either both `bind_left` or `bind_right`

Reply:
Also, switching the direction of round iteration should probably affect how the evaluation point is constructed and by implication the Lagrange kernels as well.